docs/docs/reference/project-files/dashboards.md

```diff
@@ -43,3 +43,9 @@ _**`measures`**_ — numeric [aggregates](/develop/metrics-dashboard#measures) o
       - _`currency_usd`_ —  output rounded to 2 decimal points prepended with a dollar sign
       - _`percentage`_ — output transformed from a rate to a percentage appended with a percentage sign
       - _`comma_separators`_ — output transformed to decimal formal with commas every 3 digits
+
+_**`policy`**_ - Optionally define granular access policies for the dashboard. See [Secure: Dashboards](../../develop/policies) for more details.
+  - _**`has_access`**_ - Condition to check if user has access to the dashboard. If not defined, it will resolve to `false` meaning dashboard won't be accessible to anyone. _(optional)_
+  - _**`filter`**_ - Condition to filter the data as per user/custom attributes. It needs to be a valid SQL `WHERE` clause. It will be injected in the queries after resolving the user attributes templating. _(optional)_
+  - _**`exclude`**_ - List of columns to exclude from the dashboard. Each column can have an `if` condition to check if the column should be excluded or not. If the condition is not defined, it's an error. If the condition is defined, it will be resolved after resolving the user attributes templating. If `exclude` is defined all other columns are included. _(optional)_
+  - _**`include`**_ - List of columns to include in the dashboard. Each column can have an `if` condition to check if the column should be included or not. If the condition is not defined, it's an error. If the condition is defined, it will be resolved after resolving the user attributes templating. If `include` is defined all other columns are excluded. _(optional)_
```

Comment:
1. It would be nice to have sub-bullets here for each allowed nested key, like for `measures` and others
2. Also link to the syntax for the `if` fields